hr_timesheet_hours_month/models/project_task.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    planned_hours_year = fields.Boolean(default=False)
+
+    remaining_hours_month = fields.Float(compute='_compute_remaining_hours_month')
```

Comment:
This field should be `compute_sudo=True` (see https://www.odoo.com/documentation/15.0/developer/reference/backend/orm.html#fields), as is non-stored. Users that can't access other users' timesheets can't properly calculate if not.